third_party/cpplint.py

```diff
@@ -5370,10 +5370,8 @@ def ExpectingFunctionArgs(clean_lines, linenum):
     )
 
 _HEADERS_MAYBE_TEMPLATES = (
-    ('<algorithm>', ('copy', 'max', 'min', 'min_element', 'sort',
-                     'transform',
-                    )),
-    ('<utility>', ('forward', 'make_pair', 'move', 'swap')),
+    ('<algorithm>', ('min_element',)),
```

Comment:
Before anyone asks: the trailing comma _is_ necessary. I don't know why, but without it cpplint produces some really weird warnings.

Reply:
Because in python, tuple is not a thin between parentheses but a thing separated by commas. So one element tuple is: `elem,` and not an `elem`.